profiles/local-frontend-and-api-with-identity.js

```diff
@@ -106,15 +106,13 @@ const authPlugin = (req, res, target) => {
 
 
 
-routes[`/beta/${SECTION}/${APP_ID}`] = { host: `http://localhost:${FRONTEND_PORT}` };
-routes[`/${SECTION}/${APP_ID}`]      = { host: `http://localhost:${FRONTEND_PORT}` };
-routes[`/beta/apps/${APP_ID}`]       = { host: `http://localhost:${FRONTEND_PORT}` };
-routes[`/apps/${APP_ID}`]            = { host: `http://localhost:${FRONTEND_PORT}` };
-routes[`/api/${APP_ID}`]             = { host: `http://localhost:${API_PORT}` };
+routes[`/beta/${SECTION}/${APP_ID}`] = { host: `http://frontend:${FRONTEND_PORT}` };
```

Comment:
Is it possible to set default values for these? I think we'd want this to work also without compose.